scripts/import/import_clinvar_xml

```diff
@@ -605,6 +608,24 @@ sub get_citations{
       }
     }
   }
+
+  my $observed_in_ca = to_array($structure_ca);
+  foreach my $observed_in (@{$observed_in_ca}) {
+
+    my $observed_data = to_array($observed_in->{ClinicalSignificance});
+    foreach my $obs(@{$observed_data}){
```

Comment:
This provides the results we need, but leaves us with duplicated code. 
It would be neater to call the sub twice with a key to say what field to extract data from, then merge results to avoid the duplication.

Something like 
  $record{citations} = get_citations($set->{ReferenceClinVarAssertion}->{ObservedIn}, "ReferenceClinVarAssertion");
  $extra_citations = get_citations($set->{ClinVarAssertion}, ,"ClinVarAssertion");
push (@{$record{citations}} , @{$extra_citations})